client/src/app/shared/components/data-table/data-table.component.html

```diff
@@ -1,5 +1,8 @@
-<ng-container *ngIf="dataSource || (rows && (rows.length > 0))">
+<p *ngIf="initialLoading">Loading...</p>
+
+<ng-container *ngIf="dataSource.page$ | async as page">
   <ngx-datatable #datatable
+                 *ngIf="!initialLoading && (rows && (rows.length > 0))"
                  class="material expandable"
```

Comment:
I think we don't need an extra initialLoading variable. It could just check truthness of rows?

Reply:
Yeah good shout, I will change that